src/components/DropZone/README.md

```diff
@@ -346,53 +330,50 @@ class DropZoneExample extends React.Component {
 Use to allow merchants to upload files in a wider area than the visible drop zone.
 
 ```jsx
-class DropZoneExample extends React.Component {
-  state = {
-    files: [],
-  };
-
-  render() {
-    const {files} = this.state;
-    const validImageTypes = ['image/gif', 'image/jpeg', 'image/png'];
-
-    const fileUpload = !files.length && <DropZone.FileUpload />;
-    const uploadedFiles = files.length > 0 && (
-      <Stack vertical>
-        {files.map((file, index) => (
-          <Stack alignment="center" key={index}>
-            <Thumbnail
-              size="small"
-              alt={file.name}
-              source={
-                validImageTypes.indexOf(file.type) > 0
-                  ? window.URL.createObjectURL(file)
-                  : 'https://cdn.shopify.com/s/files/1/0757/9955/files/New_Post.png?12678548500147524304'
-              }
-            />
-            <div>
-              {file.name} <Caption>{file.size} bytes</Caption>
-            </div>
-          </Stack>
-        ))}
-      </Stack>
-    );
-
-    return (
-      <DropZone
-        outline={false}
-        onDrop={(files) => {
-          this.setState({files: [...this.state.files, ...files]});
-        }}
-      >
-        <Card sectioned>
-          <DropZone onClick={() => {}}>
-            {uploadedFiles}
-            {fileUpload}
-          </DropZone>
-        </Card>
-      </DropZone>
-    );
-  }
+function NestedDropZoneExample() {
+  const [files, setFiles] = useState([]);
+
+  const handleDropZoneDrop = useCallback(
+    (dropFiles, _acceptedFiles, _rejectedFiles) =>
```

Comment:
If we never use `_acceptedFiles` or `_rejectedFiles` do we need to mention them? There's no other parameter after them that we need

Reply:
I think it'd be nice to keep them in, so consumers know the parameters are available without skimming the docs